### ISSUE TITLE
Long description is absent 

### ISSUE THREAD

**Issue by kurakin-oleksandr** [fesor/gulp-deb#6]
Hello, I just found that template does not contain long description. And possible problem is [here](https://github.com/fesor/gulp-deb/blob/master/index.js#L131):

``` javascript
lodash.template([
   ...
    'Description: <%= short_description %>',
    ' <%= short_description %>'
  ].join('\n') + '\n')(options);
```

**Comment by fesor** [comment]
Thanks for finding, Will fix it today. Or you are welcome to send PR ;)

**Comment by kurakin-oleksandr** [comment]
Sorry, don't have time for PR now

**Comment by fesor** [comment]
Implemented in #7
